### DISCUSSION TITLE
Notification when specific loot is about to despawn

### DISCUSSION THREAD

**Discussion by Crazyhalo54** [runelite/runelite#18462] [Ideas]
When AFKing content (slayer for example), I usually want to wait until the first drop of a specific item (like dragon bones) is about to despawn before running around a picking them all up. This allows me to afk for the standard 2 minute despawn timer for loot before having to pickup everything. This also helps with doing damage while waiting vs. picking up items after every kill.

Could the "Ground Items" plugin notify when a specific or highlighted item is about to despawn (or based on threshold set by the user)? I've tried to use the Clue Juggling Timer, but this only applies to items picked up and subsequently dropped again. I would like for this to work on loot specifically.

**Comment by SeraphShroud** [comment]
I would love this addition as well. It would be great to have this notify on the tier level as well. Also if there could be an option to set the notification when it's takeable by other players (1min) that would also be great.
